app/javascript/components/patient/report/ReportCompleted.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Card, Form } from 'react-bootstrap';
+
+class ReportCompleted extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <Card className="mx-0 card-square align-item-center">
+        <Card.Header className="text-center h4">{this.props.translations[this.props.lang]['web']['title']}</Card.Header>
+        <Card.Body className="text-center">
+          <Form.Label className="text-center pt-1">
+            <b>{this.props.translations[this.props.lang]['web']['thank-you']}</b>
+          </Form.Label>
+          <br />
```

Comment:
What's up with these `<br/>`s?  HTML line breaks always come up as 🚩s to me, unless there's a very good reason for them. Styling should be done semantically.

Reply:
All I did to this file was rename and move it, so the file was already structured that way. Not sure why but because this happens elsewhere too modifying it would be out of scope for this task